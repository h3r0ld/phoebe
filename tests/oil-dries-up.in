reset
end-turn
end-turn
end-turn
end-turn
end-turn
end-turn
end-turn
end-turn