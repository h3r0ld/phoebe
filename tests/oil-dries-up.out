Player 0: Cell(22,8) Speed: 2 CanChangeDirection: true
Player 1: Cell(22,10) Speed: 2 CanChangeDirection: true
Player 0: Cell(22,8) Speed: 2 CanChangeDirection: true
Cell(20,10) GameObject: null
Player 1: Cell(22,10) Speed: 2 CanChangeDirection: true
